cardano-db-sync/src/Cardano/DbSync/Api.hs

```diff
@@ -199,8 +199,8 @@ getPrunes :: SyncEnv -> Bool
 getPrunes = do
   DB.pcmPruneTxOut . getPruneConsume
 
-fullInsertOptions :: Bool -> InsertOptions
-fullInsertOptions useLedger = InsertOptions True useLedger True True True True True True True
+fullInsertOptions :: Bool -> [Text] -> InsertOptions
+fullInsertOptions useLedger keepTxMDNames = InsertOptions True useLedger True True True True keepTxMDNames True True True
```

Comment:
Personally, I would prefer to see the record form of the constructor here, it's a lot easier to follow when there are many arguments of the same type.